tests/ut-ncnnapi.cc

```diff
@@ -312,7 +320,18 @@ TEST(ncnnapi, ocr)
         + ocr_repo
         + "\"},\"parameters\":{\"input\":{\"connector\":\"image\",\"ctc\":"
           "true, \"height\":136,\"width\":220},\"mllib\":{\"nclasses\":69}}}";
-  std::string joutstr = japi.jrender(japi.service_create(sname, jstr));
+
+  auto service_create = dd::DTO::ServiceCreate::createShared();
```

Comment:
Is there a way of creating the DTO object directly from the JSON string ?

Reply:
Yes, and that's a good idea to do it in tests, it will ensure the current json string still work as-is.